dace/libraries/blas/include/dace_cublas.h

```diff
@@ -134,18 +129,28 @@ class _CublasConstants {
   double* double_pone_;
   cuComplex* complex64_pone_;
   cuDoubleComplex* complex128_pone_;
+  void* custom_alpha_;
+  void* custom_beta_;
 };
 
-class CublasConstants {
+class CublasHandle {
```

Comment:
Can you pls add a small comment about the purpose of this class?

Reply:
Done.